lib/spack/spack/test/bindist.py

```diff
@@ -103,9 +103,6 @@ def default_config(
     # This is essential, otherwise the cache will create weird side effects
     # that will compromise subsequent tests if compilers.yaml is modified
     monkeypatch.setattr(spack.compilers, '_cache_config_file', [])
-    njobs = spack.config.get('config:build_jobs')
-    if not njobs:
-        spack.config.set('config:build_jobs', 4, scope='user')
```

Comment:
I thought this was redundant, because `config:build_jobs` is always populated https://github.com/spack/spack/blob/develop/lib/spack/spack/config.py#L103-L117